### PR TITLE
kubectl-cheatsheet - patch, added example for add

### PR DESCRIPTION
**Pull request by paoloantinori**
Added an example showing how to add an element to a list, using `op=add` operation.

My example is specifically taken from an OpenShift `DeploymentConfig` resource, but the idea is more generic.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4546)

### CODE REVIEW

**Comment by chenopis** [comment]
Deploy preview ready!

Built with commit 90db9209b67e0adde71576c32c4ea3f60ab2df09

https://deploy-preview-4546--kubernetes-io-master-staging.netlify.com

**Comment by paoloantinori** [comment]
just signed the CLA

**Comment by steveperry-53** [comment]
@mengqiy @liggitt @pwittrock @erictune @krousey This is a fairly small addition. Could one of you take a quick look and give a tech approval? Thanks.

**Comment by foxish** [comment]
@kubernetes/sig-cli-maintainers PTAL

File: docs/user-guide/kubectl-cheatsheet.md [hunk 1]
```diff
@@ -171,6 +171,9 @@ $ kubectl patch pod valid-pod --type='json' -p='[{"op": "replace", "path": "/spe
 
 # Disable a deployment livenessProbe using a json patch with positional arrays
 $ kubectl patch deployment valid-deployment  --type json   -p='[{"op": "remove", "path": "/spec/template/spec/containers/0/livenessProbe"}]'
+
+# Add a new element to a positional array 
+$ kubectl patch dc my_deployment_config --type='json' -p='[{"op": "add", "path": "/spec/template/spec/containers/1", "value": {"image": "my_new_image", "name": "my_new_image"} }]'
```
Comment (mengqiy) [inline docs/user-guide/kubectl-cheatsheet.md]: k8s doesn't have resource `deploymentconfig` ATM, so I suggest change it to deployment.

**Comment by steveperry-53** [comment]
@mengqiy Thanks for the review. @paoloantinori Can you make the change to Deployment? Thanks.

**Comment by paoloantinori** [comment]
Added a commit with the requested change.

**Comment by steveperry-53** [comment]
@mengqiy Can you take a look at the revisions, and approve if they seem OK?
@paoloantinori Please delete line 161 saying Patch a resource(s) with a strategic merge patch.

**Comment by paoloantinori** [comment]
@steveperry-53 done

**Pull request merged**